apps/backoffice-v2/src/lib/blocks/components/PDFViewerCell/PDFViewer.tsx

```diff
@@ -0,0 +1,7 @@
+import { TPDFViewerCell } from '@/lib/blocks/components/PDFViewerCell/interfaces';
+import { FunctionComponent } from 'react';
+
+export const PDFViewerCell: FunctionComponent<TPDFViewerCell> = ({ props, value }) => {
```

Comment:
Please make sure to use the helpers extracted from `@ballerine/blocks`. Let's sync on this.